### DISCUSSION TITLE
Allow plugins to be free for free Solo edition Craft CMS installs

### DISCUSSION THREAD

**Discussion by khalwat** [craftcms/console#112] [Craft Console]
I've long wanted to make all of the nystudio107.com plugins available for free for people who are running free Solo edition Craft CMS installs.

The rationale is that if it's a Solo edition site, it's usually either a freelancer or agency's own website, or it's a smaller client site with little budget.

Currently, there's no way I'm aware of that a license can be tied to a particular edition of Craft, so I'd like to propose that just as Craft CMS has different pricing models for different editions of Craft CMS, that plugin developers can also choose different pricing tier depending on the Craft CMS edition.

This would be opt-in, so all existing plugins would still work as they do now. But you could **add** a plugin edition and tie it to a specific Craft CMS edition.

That way developers like me who'd like their plugins to be free for Solo edition Craft CMS sites (or discounted perhaps) could do that.

**Comment by tommysvr** [comment]
This would be a great option. I work with a few local non-profit, small community clubs who typically only need a Solo license. To be able to offer them some of the near-essential plugins at a heavily discounted (or free!) rate would be super empowering for those organisations.

**Comment by zizther** [comment]
+1

**Comment by thupsi** [comment]
Great idea! +1
